host-bin/edit-chroot

```diff
@@ -335,15 +363,34 @@ for NAME in "$@"; do
                 dest="$dest.gz"
             fi
         fi
+        # If we're writing to a fat32 filesystem, split the file at 4GB chunks
+        if ! df -T "`getmountpoint "$dest"`" | awk '$2~"^v?fat"{exit 1}'; then
+            SPLIT="${SPLIT:-4GB}"
+        fi
+        if [ -n "$SPLIT" ]; then
+            tardest="`mktemp -u --tmpdir=/tmp 'crouton-backup.XXX'`"
```

Comment:
unsafe (okay, the likelyhood of something bad happening is low, but, still). Better create a temp directory and put the fifo in it.